src/layout/DesignSection/DesignSection.svelte

```diff
@@ -1,14 +1,17 @@
 <script lang="ts">
 	import { defaultI18nValues, HeaderChip, PageSection } from "$lib";
 	import { Button } from "fluent-svelte";
+	import { onMount } from "svelte";
 	import { _ } from "svelte-i18n";
 	let scrollY: number;
-</script>
 
-<svelte:window
-	on:scroll={() =>
-		window.requestAnimationFrame(() => (scrollY = window.scrollY))}
-/>
+	onMount(() => {
+		const body = document.getElementsByTagName("body")[0];
+		body.addEventListener("scroll", () => {
+			scrollY = body.scrollTop;
+		});
+	});
```

Comment:
Since you're using `onMount` here, you should return a function to be called when the component is destroyed, like so:


```suggestion
	onMount(() => {
		const body = document.getElementsByTagName("body")[0];
    const handleScroll = () => {
			scrollY = body.scrollTop;
		}
		body.addEventListener("scroll", handleScroll);

    return () => body.removeEventListener("scroll", handleScroll)
	});
```

the docs for the `removeEventListener` function are available here: https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/removeEventListener

Reply:
Should we also remove the event listener for the theme change event in that componenet when it gets destroyed?

Reply:
Yeah, sure. it won't make too much of a difference, but it's probably better convention.